web-gateway/conf/application.conf

```diff
@@ -11,4 +11,13 @@ lagom.play {
       path-regex = "(?!/api/).*"
     }
   ]
+}
+
+lagom.circuit-breaker {
+  default {
+    enabled = on
+    max-failures = 10
+    call-timeout = 10s
+    reset-timeout = 15s
+  }
```

Comment:
Hi @tommpy, the values you suggest for the `lagom.circuit-breaker.default` are equal to Lagom's defaults: https://github.com/lagom/lagom/blob/f62df3283a3006b2c1bf13c4ca07e53acce277b8/service/core/client/src/main/resources/reference.conf#L6-L24. 

I think the solution to slow downstream services bootup could be to remove the setting on line 3 which makes the circuit breakers act a bit more aggressively than defaults.